stock_picking_show_return/README.rst

```diff
@@ -0,0 +1,50 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+==============================
+Show returns on stock pickings
+==============================
+
+This module enables one tab in the pickings that contains returns made for that 
```

Comment:
This module adds one tab in the pickings form view to display the returns pickings related to the current one.